### ISSUE TITLE
Not an issue but rather a question (Can we use it on Linux ?)

### ISSUE THREAD

**Issue by Electrodog** [UltimateFishbot/UltimateFishbot#163]
I'm totally new to Linux, I finally got Fedora to install wow through Wine, I then tried using the bot by installing Net Framework on Fedora but it just doesn't work. 
So is there something else I need to install in order to make it work on Linux ?

**Comment by daniel-widrick** [comment]
No. Uses winforms among other things. You are welcome to try playing with wine, core lib and everything else but it is all officially unsupported.
